srcalc/src/main/java/gov/va/med/srcalc/web/view/VariableEntry.java

```diff
@@ -45,8 +46,23 @@ public VariableEntry(final Collection<? extends Variable> variables)
     }
     
     /**
+     * Constructs a {@link VariableEntry} with reasonable defaults for some of
+     * the variables and automatically retrieved values for other variables.
+     */
+    public VariableEntry(final Calculation calculation)
+    {
+        final DefaultValueGenerator dvg = new DefaultValueGenerator();
```

Comment:
Code duplication. I suggest making this a factory method instead for clarity, something like `VariableEntry.withInitialValues()`. Also please add a unit test for this. Changing the parameters to a Variable collection and a Patient (instead of a Calculation) makes the interface more flexible and will facilitate writing the test.